src/Tools/dotnet-counters/Program.cs

```diff
@@ -32,17 +33,18 @@ internal class Program
 
         private static Option RefreshIntervalOption() =>
             new Option(
-                new[] { "-r", "--interval" }, 
+                new[] { "-r", "--refresh-interval" }, 
```

Comment:
It would be nice to get rid of the "-r" (the dotnet CLI folks recommended we use single letter sparingly for only very common things)

Reply:
Ah, I see. Will get rid of it then. Thanks!